tools/backups2datalad.py

```diff
@@ -100,25 +117,48 @@ def run(self, dandisets=()):
                 if not ds.is_installed():
                     log.info("Creating Datalad dataset")
                     ds.create(cfg_proc="text2git")
-                with dandi_logging(dsdir):
-                    # dandi_logging() creates a file in the dataset directory,
-                    # which makes ds.create() fail if dandi_logging() is run
-                    # first.
-                    if self.sync_dataset(did, ds):
-                        log.info("Creating GitHub sibling for %s", ds.pathobj.name)
-                        ds.create_sibling_github(
-                            reponame=ds.pathobj.name,
-                            existing="skip",
-                            name="github",
-                            access_protocol="ssh",
-                            github_organization=self.gh_org,
+                    if self.backup_remote is not None:
+                        ds.repo.init_remote(
+                            self.backup_remote,
+                            [],
+                            type="rclone",
+                            external=True,
+                            config={
+                                "chunk": "1GB",
+                                "target": self.backup_remote,  # I made them matching
+                                "prefix": "dandi-dandisets/annexstore",
+                                "embedcreds": "no",
+                                "uuid": "727f466f-60c3-4778-90b2-b2332856c2f8"
```

Comment:
Just a mental note to myself that I forgot about this uuid. Wouldn't be appropriate for some other of beyond testing... Will need to become an option